right-for-life_front/src/components/Header/Header.jsx

```diff
@@ -9,12 +10,38 @@ export const Header = () => {
 					<li className="mx-4 cursor-pointer ">
```

Comment:
Cursor becomes pointer when over Logo but click doesn't lead anywhere. Maybe it should point to root?

Reply:
Alright